package.json

```diff
@@ -7,8 +7,10 @@
     "develop": "gulp develop",
     "build-dist-js": "gulp build-dist-js",
     "test": "gulp test-coverage",
-    "lint": "eslint src example --ext .jsx --ext .js"
-},
+    "lint": "eslint src example --ext .jsx --ext .js",
+    "prepublish": "gulp build-dist-scss"
+  },
+  "style": "./dist/css/react-calendar.css",
```

Comment:
This would permit the use of `import` statements from CSS files using libraries like npm-css, npm-less, rework-css, postcss-import. So, it requires the user to explicitly add something like:
```css
@import "react-daterange-picker";

body {
  color: red;
}
```
re: webpack: I think webpack requires you explicitly require the CSS file itself, it won't auto-detect the `style` entry in `package.json`, as best I can tell.
re: major release: The default behavior varies by `@import` library, but because there was no `index.css` in the module, I don't think it can break the existing behavior (which just wouldn't have let people compile at all). So I think a minor release is sufficient because the addition is not backwards incompatible.